package.json

```diff
@@ -43,7 +45,7 @@
   "scripts": {
     "lint": "eslint --ext .js index.js",
     "lint-fix": "eslint --fix --ext .js index.js",
-    "test": "npm run lint",
```

Comment:
test commande should run all test lint + jest.
Can you di a specific commande for jest test and modify test commande to run jest and test npm commande please ?